### ISSUE TITLE
Simplify language assignment?

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#755]
Although it's not clear from the specification, it turns out that language containers are handled correctly when they're just strings with no language associated:

```
{
  "@context":
  {"occupation": {"@id": "ex:occupation", "@container": "@language"}},
  "occupation": "Nindža"
}
```

Works at the same time as:

```
{
  "@context":
  {"occupation": {"@id": "ex:occupation", "@container": "@language"}},
  "occupation": {"en": "ninja"}
}
```

However multiple are not. This fails:

```
{
  "@context":
  {"occupation": {"@id": "ex:occupation", "@container": "@language"}},
  "occupation": [{"en": "ninja"}, "ninjas"]
}
```

And has to be, instead:

```
{
  "@context":
  {"occupation": {"@id": "ex:occupation", "@container": "@language"}},
  "occupation": {"en": "ninja", "": "ninjas"}
}
```

This seems like an improvement over the lists of @language/@value pairs?

**Comment by azaroth42** [comment]
Hrm. I think this is a bug in the JSON-LD algorithm. The spec is clearer in the language map definition that it must be a node object, not a string.

**Comment by azaroth42** [comment]
And reply from @gkellogg:
> Hi Rob, you’re correct that _to be a language map_, the term must be defined using @container: @language, _and_, the value must be an object with keys as described. However, the JSON-LD expansion algorithm allows the same key to be used in ways that it is not interpreted as a language map, notably those you mention

So we could do this in 3.0 (`defer`ed until then anyway)

**Comment by azaroth42** [comment]
Hmm... note https://github.com/digitalbazaar/jsonld.js/issues/151#issuecomment-249190822

> "" cannot be used as a key in certain languages (e.g. PHP)

There is an "undefined" language key though:  UND

**Comment by azaroth42** [comment]
NB ... the PHP issue was resolved in 7.1, opening the door for the preferable, round-trippable {"": "un-languaged text"} pattern. 

Propose that for 3.0 we should go ahead with this revised pattern for i18n.

**Comment by azaroth42** [comment]
Dear IIIFers ... please can you :+1: this proposal:
  https://github.com/json-ld/json-ld.org/issues/480#issuecomment-299011791

**Comment by azaroth42** [comment]
Discussion 08/30:  Everyone likes the pattern, and assuming that everything goes well with standardization and implementations of it, we'll adopt it.

**Comment by azaroth42** [comment]
Discussion at Getty AV:
* Consistency is better than inconsistency (or many many options). Every exception is an additional barrier to entry.
* But raises the very initial barrier (e.g. need to understand language map to understand `label`)
* Somewhat more complex for content creators, as there is additional constraints on how simple labels must be provided.
* Nicer for client developers, which might encourage more clients.  
* OTOH, there are currently many more developers of content publishing rather than clients.
* Good for *International* Image Interoperability. We're saying that internationalization is a good thing, which is valuable.

**Comment by zimeon** [comment]
:+1: agreed at Toronto WG meeting. 

Consensus was that the regularity and promotion of internationalization outweigh concerns over slight increase in complexity for simple case. It was noted that the 3.0 spec should change the example to make clear why one would have two values for one language (current example lists "and a second description" for the second value). It should also be noted that use of `@none` and hyphenated language values (e.g. `ar-latn`) means that client implementations should be aware of these as possible key values in the map.

**Comment by azaroth42** [comment]
Closed by #1298.
